Angular/src/app/add-role-modal/add-role-modal.component.spec.ts

```diff
@@ -1,14 +1,21 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { AddRoleModalComponent } from './add-role-modal.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { DataService } from '../_services/data.service';
 
 describe('AddRoleModalComponent', () => {
   let component: AddRoleModalComponent;
   let fixture: ComponentFixture<AddRoleModalComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ AddRoleModalComponent ]
+      declarations: [ AddRoleModalComponent ],
+      imports: [ ReactiveFormsModule ],
+      providers: [ NgbActiveModal,
+                   DataService
+                 ]
```

Comment:
same here

Reply:
I prefer the formatting like this: https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17